lib/matplotlib/axes/_base.py

```diff
@@ -1011,7 +1014,7 @@ def cla(self):
         else:
             self.yaxis._set_scale('linear')
             try:
-                self.set_ylim(0, 1)
+                self.set_ylim(0, 1, _converter=False)
```

Comment:
Can kwargs be private? (I've never seen this before so I have no idea)

Reply:
From a philosophical standpoint, nothing can be private in the panopticon that is Python symbol visibility, but [here is an example from pytz](https://github.com/stub42/pytz/blob/b4dfd2cc2099185d8bf507592bc4ff8d50cfaa6c/src/pytz/__init__.py#L425) of a private kwarg (though I think popping it from `**kwargs` so it doesn't show up under inspection is a better course to take).

Reply:
I think it is pretty clear that a kwarg with an underscore is intended as private.

Reply:
Maybe the better question is, will an underscored kwarg appear in the documentation? (I'm not convinced in this case that the kwarg is needed anyway)

Reply:
@dstansby Looks to me like the way it's implemented, it's just being pulled from the `kwargs` dictionary, it's not an actual declared keyword argument.

Reply:
Just to justify this kwarg - the point of this PR is that the first time you conver units on an axis, the converter gets set.  This happens on `set_xticks`, `set_xlims` or any of the plotting calls. 

`cla` calls `set_xlim` but its not doing so because it has any data that might have units.  So this kwarg is just an easy way to say "set the limit, but don't assign the units because this is just a placeholder".  

We could, alternately, make a new method (`_set_xlim_placeholder`) that did the same thing, if that is better than a hidden kwarg?

Reply:
I think either way is fine.